cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/encryption/EncryptedImageCopyService.java

```diff
@@ -78,8 +78,9 @@
                 .collect(Collectors.toMap(Entry::getKey, e -> e.getValue().getImageId()));
         if (!imageIdByGroupName.isEmpty()) {
             Collection<String> imageIds = new HashSet<>(imageIdByGroupName.values());
-            LOGGER.debug("Start polling the availability of the created AMIs: '{}'", String.join(",", imageIds));
+            LOGGER.debug("Start polling the availability of the created encrypted AMIs: '{}'", String.join(",", imageIds));
             checkBooleanPollTask(awsPollTaskFactory.newAMICopyStatusCheckerTask(ac, imageIds, client));
+            LOGGER.debug("All created encrypted AMIs are available: '{}'", String.join(",", imageIds));
```

Comment:
I think it could be on info level, as it is an important info. This log should have been added by me years ago :)

Reply:
Done.